test/test_utils.py

```diff
@@ -6,6 +6,43 @@
 seed = 42
 
 
+def test_quantile_aggregation():
+    """
+    This function tests the application of the quantile aggregation method
+    """
+    col = np.arange(11)
+    p_values = np.tile(col, (10, 1)).T / 100
+    gamma_min = 0.05
+
+    assert_array_almost_equal(0.1 * quantile_aggregation(p_values, 0.1), [0.01] * 10)
+    assert_array_almost_equal(
+        0.1
+        * quantile_aggregation(p_values, 0.1, adaptive=True, gamma_min=gamma_min)
+        / (1 - np.log(gamma_min)),
+        [0.01] * 10,
+    )
+    assert_array_almost_equal(0.3 * quantile_aggregation(p_values, 0.3), [0.03] * 10)
+    assert_array_almost_equal(
+        0.3
+        * quantile_aggregation(p_values, 0.3, adaptive=True, gamma_min=gamma_min)
+        / (1 - np.log(gamma_min)),
+        [0.03] * 10,
+    )
+    assert_array_almost_equal(0.5 * quantile_aggregation(p_values, 0.5), [0.05] * 10)
+    assert_array_almost_equal(
+        0.5
+        * quantile_aggregation(p_values, 0.5, adaptive=True, gamma_min=gamma_min)
+        / (1 - np.log(gamma_min)),
+        [0.05] * 10,
+    )
+
+    # One p-value within the quantile aggregation method
+    p_values = np.array([0.0])
+
+    assert quantile_aggregation(p_values) == 0.0
+    assert quantile_aggregation(p_values, adaptive=True) == 0.0
```

Comment:
This test is a bit artificial. I suggest to check that the aggregated p-value is in [0, 1] and put multiple p-value in the p-values array.

Reply:
I added these last two tests because there is a condition in the quantile_aggregation about how to  manage an array of one pvalue.